frontend/src/types/InterimAgencyInfo.ts

```diff
@@ -0,0 +1,6 @@
+type InterimAgencyInfo = {
+  name: string;
+  number: string;
```

Comment:
Why use number as string here?

Reply:
I don't know exactly how we will receive the data from the API so I just put a "generic" type here